src/Saas.Admin/Saas.Admin.Service/Controllers/TenantsController.cs

```diff
@@ -208,7 +212,7 @@ public async Task<IActionResult> DeleteTenant(Guid tenantId)
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult<IEnumerable<string>>> GetTenantUsers(Guid tenantId)
+    public async Task<ActionResult<IEnumerable<string>>> GetTenantUsers(string tenantId)
```

Comment:
Why the change? for model binding?

Reply:
@asanjabi and I went back and forth on this a bit. We ultimately decided it was better to use strings across the board for IDs incase the user wanted to use something other than GUIDs for tenantIDs.